boards/xtensa/m5stickc_plus/doc/index.rst

```diff
@@ -0,0 +1,137 @@
++==================+=========================================================================+
+| 32.768 kHz RTC   | External precision 32.768 kHz crystal oscillator serves as a clock with |
+|                  | low-power consumption while the chip is in Deep-sleep mode.             |
++------------------+-------------------------------------------------------------------------+
+| ESP32-PICO-D4    | This `ESP32-PICO-D4`_ module provides complete Wi-Fi and Bluetooth      |
+| module           | functionalities and integrates a 4-MB SPI flash.                        |
++------------------+-------------------------------------------------------------------------+
+| Diagnostic LED   | One user LED connected to the GPIO pin.                                 |
++------------------+-------------------------------------------------------------------------+
+| USB Port         | USB interface. Power supply for the board as well as the                |
+|                  | communication interface between a computer and the board.               |
+|                  | Contains: TypeC x 1, GROVE(I2C+I/O+UART) x 1                            |
++------------------+-------------------------------------------------------------------------+
+| Power Switch     | Power on/off button.                                                    |
++------------------+-------------------------------------------------------------------------+
+| A/B user buttons | Two push buttons intended for any user use.                             |
++------------------+-------------------------------------------------------------------------+
+| LCD screen       | Built-in LCD TFT display \(`ST7789v2`_, 1.14", 135x240 px\) controlled  |
+|                  | by the SPI interface                                                    |
++------------------+-------------------------------------------------------------------------+
+| MPU-6886         | The `MPU-6886`_ is a 6-axis MotionTracking device that combines a       |
+|                  | 3-axis gyroscope and a 3-axis accelerometer.                            |
++------------------+-------------------------------------------------------------------------+
+| Built-in         | The `SPM-1423`_ I2S driven microphone.                                  |
+| microphone       |                                                                         |
++------------------+-------------------------------------------------------------------------+
+
+
+Start Application Development
+*****************************
+
+Before powering up your M5StickC PLUS, please make sure that the board is in good
+condition with no obvious signs of damage.
+
+System requirements
+===================
+
+Prerequisites
+-------------
+
+Espressif HAL requires binary blobs in order work. The west extension below performs the required
+synchronization to clone, checkout and pull the submodules:
+
+.. code-block:: console
+
+   west espressif update
```

Comment:
Would you update this documentation? This command is no longer available. Please, check `esp32` board documentation as reference.